Makefile

```diff
@@ -48,18 +50,18 @@ format:
 	flutter format packages/bugsnag_flutter example features/fixtures/app
```

Comment:
Does this instance of `flutter` also need updating?

Reply:
Not sure why this was missed during the find and replace.

Reply:
Looks like it still needs changing?